Runtime/BlockadeLabsSkyboxMesh.cs

```diff
@@ -22,6 +20,15 @@ public enum MeshDensity
     [ExecuteAlways, RequireComponent(typeof(MeshRenderer)), RequireComponent(typeof(MeshFilter))]
     public class BlockadeLabsSkyboxMesh : MonoBehaviour
     {
+        [SerializeField]
+        private SkyboxAI _skyboxMetadata;
+
+        public SkyboxAI SkyboxAsset
+        {
+            get => _skyboxMetadata;
+            internal set => _skyboxMetadata = value;
```

Comment:
still think this needs to be public, and also do the work to update the material on the mesh renderer